Makefile

```diff
@@ -10,6 +20,34 @@ help:
 fmt:
 	for i in *.json ; do jq --indent 2 -M . "$${i}" > xx && cat xx > "$${i}" && rm xx ; done
 
+docs: output/docs.pdf output/docs.html
+.PHONY: docs
+
+output/docs.pdf: $(DOC_FILES)
+	@mkdir -p output/ && \
+	$(DOCKER) run \
+	-it \
+	--rm \
+	-v $(shell pwd)/:/input/:ro \
+	-v $(shell pwd)/output/:/output/ \
+	-u $(shell id -u) \
+	vbatts/pandoc -f markdown_github -t latex -o /output/docs.pdf $(patsubst %,/input/%,$(DOC_FILES)) && \
+	ls -sh $(shell readlink -f $@)
+
+output/docs.html: $(DOC_FILES)
+	@mkdir -p output/ && \
+	$(DOCKER) run \
+	-it \
+	--rm \
+	-v $(shell pwd)/:/input/:ro \
+	-v $(shell pwd)/output/:/output/ \
+	-u $(shell id -u) \
+	vbatts/pandoc -f markdown_github -t html5 -o /output/docs.html $(patsubst %,/input/%,$(DOC_FILES)) && \
+	ls -sh $(shell readlink -f $@)
+
+code-of-conduct.md:
+	curl -o $@ https://raw.githubusercontent.com/opencontainers/tob/d2f9d68c1332870e40693fe077d311e0742bc73d/code-of-conduct.md
```

Comment:
Maybe [float with master][1]?  While I like reproducible builds, I think “we've pulled an inappropriately modern code of conduct when rebuilding the old v0.1.0 docs” is not a big enough issue to justify bumping this commit hash.  And a more modern code of conduct will probably be an uncontroversial improvement to the old docs anyway.

[1]: https://raw.githubusercontent.com/opencontainers/tob/master/code-of-conduct.md